egret/model_library/transmission/branch.py

```diff
@@ -886,11 +961,26 @@ def declare_ineq_p_interface_bounds(model, index_set, interfaces,
 
     m.ineq_pf_interface_bounds = pe.Constraint(con_set)
 
+    pos_slacks = hasattr(model, 'pfi_slack_pos')
+    neg_slacks = hasattr(model, 'pfi_slack_neg')
```

Comment:
Same thing here - can use use an argument to the function for this?

Reply:
Changed to use a function argument similar to above.